README.md

```diff
@@ -7,7 +7,7 @@ There are two sbt plugins in this one plugin library:
 
 # Compatibility
 
-Tested and supported for sbt version 0.13.x and above.
+Tested and supported for sbt versions: 0.13.17, 1.1.6, and 1.2.1
```

Comment:
Technically this is what we test with. I'm assuming these tests reflect the other version in each minor version.